app/build.gradle

```diff
@@ -64,4 +65,14 @@ dependencies {
     // Coroutines
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutinesVersion"
+    // Retrofit 2
```

Comment:
I usually prefer leaving an empty line between import regions, in order to make it mode readable.